addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureFileInclusionScanRuleUnitTest.java

```diff
@@ -85,4 +101,141 @@ void shouldReturnExpectedExampleAlert() {
         assertThat(alert.getRisk(), is(equalTo(Alert.RISK_HIGH)));
         assertThat(alert.getConfidence(), is(equalTo(Alert.CONFIDENCE_MEDIUM)));
     }
+
+    @Test
+    void shouldSkipUrlParams() {
+        // Given
+        HttpMessage msg = createMessage("/test/");
+        NameValuePair nvp = new NameValuePair(NameValuePair.TYPE_URL_PATH, "inc", "", 0);
+        // When
+        rule.scan(msg, nvp);
```

Comment:
There's no reason to call this one directly.

Reply:
So get rid of this test?

Reply:
It should call init/scan.